config/environments/development.rb

```diff
@@ -43,4 +43,14 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    user_name: secrets.mailer_username,
```

Comment:
We usually use extended object
```suggestion
    user_name: Rails.application.secrets.mailer_username,
```